### PR TITLE
Add wxWebView::EnableBrowserAcceleratorKeys()

### PR DESCRIPTION
**Pull request by TcT2k**
The edge backend has various accelerator keys enabled per default. This includes Ctrl-P, Ctrl-F, F5 and others. In many uses of webview this might not be desirable and can be disabled.

This is currently only implemented for the Edge backend.

Raises required WebView2 SDK version from 1.0.705.50 to 1.0.864.35

### CODE REVIEW

File: include/wx/msw/private/webview_edge.h [hunk 1]
```diff
@@ -39,6 +39,7 @@ __CRT_UUID_DECL(ICoreWebView2NavigationStartingEventHandler, 0x9adbe429, 0xf36d,
 __CRT_UUID_DECL(ICoreWebView2NewWindowRequestedEventHandler, 0xd4c185fe, 0xc81c, 0x4989, 0x97,0xaf, 0x2d,0x3f,0xa7,0xab,0x56,0x51);
 __CRT_UUID_DECL(ICoreWebView2SourceChangedEventHandler, 0x3c067f9f, 0x5388, 0x4772, 0x8b,0x48, 0x79,0xf7,0xef,0x1a,0xb3,0x7c);
 __CRT_UUID_DECL(ICoreWebView2WebMessageReceivedEventHandler, 0x57213f19, 0x00e6, 0x49fa, 0x8e,0x07, 0x89,0x8e,0xa0,0x1e,0xcb,0xd2);
+__CRT_UUID_DECL(IID_ICoreWebView2Settings3, 0xfdb5ab74, 0xaf33, 0x4854, 0x84,0xf0,0x0a,0x63,0x1d,0xeb,0x5e,0xba);
```
Comment (MaartenBent) [inline include/wx/msw/private/webview_edge.h:42]:
```suggestion
__CRT_UUID_DECL(ICoreWebView2Settings3, 0xfdb5ab74, 0xaf33, 0x4854, 0x84,0xf0,0x0a,0x63,0x1d,0xeb,0x5e,0xba);
```

Comment (TcT2k) [reply include/wx/msw/private/webview_edge.h]: Good idea, will do. And my page error here shows that we should probably include it in some CI test

Comment (MaartenBent) [reply include/wx/msw/private/webview_edge.h]: Indeed, I also suggested this in https://github.com/wxWidgets/wxWidgets/issues/23066#issuecomment-1363230641. I can create a PR that adds this.

**Comment by vadz** [comment]
Looks good, thanks, will merge soon.

And, of course, testing this code in the MinGW CI builds would be useful.

**Pull request merged**